stage1/app-rm/app-rm.go

```diff
@@ -59,7 +69,31 @@ func main() {
 	}
 
 	enterCmd := stage1common.PrepareEnterCmd(false)
+	switch flagStage {
```

Comment:
I've left this as is for the moment. This an internal implementation anyway, perhaps we can make this clearer down the road if we have other similar cases to document/classify.